### ISSUE TITLE
Make switchers more accessible

### ISSUE THREAD

**Issue by jmduke** [buttondown/roadmap#3487]
Sounds like they're not keyboard-friendly:

https://mastodon.social/@weirdwriter@tweesecake.social/113182633989418329

They are implemented as radios atm, but I wonder if our wrapping it in a form tag is what's problematic?

**Comment by mary-ext** [comment]
Hrmmm, I don't think it's different from how we're using switchers in other places, but I wonder if it's because the user was expecting to Tab between options rather than use arrow keys?

**Comment by rkingett** [comment]
For some reason, they do not appear as any interactive elements, just plain text, which is why I suggested making them radio buttons or comboboxes, but upon trying the form again, the screen reader itself, NVDA, in this case, does not see them as interactive controls. Whatever control is used now, they don't convey states, so even if it works with the keyboard a screen reader user, at the moment, wouldn't be able to tell which option they selected.

**Comment by mary-ext** [comment]
@rkingett Heya, we've deployed an update that makes some changes to the switchers and after some brief tests with screen readers on my end I can confirm that they're now being read out, can you confirm that this works for you?

Thanks!

**Comment by rkingett** [comment]
Yes! the update makes them much more accessible! Is there a way to sort of track these in other menus and otherwise too, like when editing automations or editing tags? The new add payments flow after the update is far more accessible now and very keyboard friendly too!

**Comment by mary-ext** [comment]
Switcher inputs in all parts of the app, including the ones on automations, should be updated as well. Let us know if you find any other issues with it ^^ for now I'm going to consider this done
